src/app/components/main-page/main-page.component.html

```diff
@@ -0,0 +1,147 @@
+
```

Comment:
It is better to start writing from the first line :)

Reply:
done